packages/language-server/test/plugins/typescript/features/CompletionProvider.test.ts

```diff
@@ -1290,7 +1330,7 @@ describe('CompletionProviderImpl', function () {
             insertText: undefined,
             insertTextFormat: undefined,
             labelDetails: undefined,
-            commitCharacters: ['.', ',', ';', '('],
+            commitCharacters: [],
```

Comment:
This one is returned by the TypeScript. The TypeScript PR mentioned it is to prevent something like this from triggering accepting completion suggestions. It was added to VSCode first and then migrated to TypeScript. 

```ts
someFunction(`somting.`)

function someFunction(str: 'something.a', 'something.b') {}
```